models/experimental/functional_yolov4/reference/resblock.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
please generate the pytorch graph of ResBlock from here and compare to the ResBlock of the reference graph from the original implementation attached here. Also include the auto-generated TTNN graph. and lets ensure all 3 graphs match as a sanity check.

Reply:
[resblock_downsample2_pytoch.gv.pdf](https://github.com/tenstorrent/tt-metal/files/15006681/resblock_downsample2_pytoch.gv.pdf)
[resblock_downsample3_pytoch.gv.pdf](https://github.com/tenstorrent/tt-metal/files/15006682/resblock_downsample3_pytoch.gv.pdf)